sdk/servicebus/azure-servicebus/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,429 @@
+# Troubleshoot Azure Service Bus client library issues
+
+This troubleshooting guide contains instructions to diagnose frequently encountered issues while using the Azure Service Bus client library for Python.
```

Comment:
Copy over any and all relevant sections from the .NET troubleshooting guide: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.Messaging.ServiceBus/TROUBLESHOOTING.md

Given that the Python SDK is not concurrency safe, sections about concurrency should be modified or removed accordingly.

If there are .NET specific exceptions and code, replace with info relevant to the Python SDK.

Reply:
Under the "Troubleshooting receiver issues" section, note in a subheader about Message Completion:
The Pure Python AMQP implementation doesn't currently wait for dispositions from the service to acknowledge message completion.

Update this description to give a more thorough explanation.